elife/config/etc-init.d-goaws

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
```

Comment:
this should be ported to a `etc-init-goaws.conf` Upstart script like we do for long-running processes, that should make it much shorter?